code/modules/clothing/suits/armor.dm

```diff
@@ -95,6 +95,7 @@
 /obj/item/clothing/suit/armor/defender
 	name = "Defender MK2"
 	desc = "A New-Russia combat armor vest with a flat-green camoflauge. It uses ceramic plates to grant the wearer extremely effective protection against ballistic weaponry and moderate protection from explosions."
+	slowdown = 0.5

```

Comment:
The armor is somewhat better than the bulletproof vest yet it lets you run just as fast, so this is meant to balance it a bit more.